shell.nix

```diff
@@ -3,6 +3,7 @@
   sha256 = "sha256:1k5nvn2yzw370cqsfh62lncsgydq2qkbjrx34cprzf0k6b93v7ch";
 }) {} }:
 
+
```

Comment:
Unnecessary newline.

Reply:
Addressed in [21f2635](https://github.com/status-im/nwaku/pull/1190/commits/21f263591d0735a6e901978054ba3e2132ebf088)